test/ecto/query/inspect_test.exs

```diff
@@ -245,33 +254,51 @@ defmodule Ecto.Query.InspectTest do
   end
 
   test "to_string all" do
```

Comment:
We don't need to change this particular test. Testing one of combinations is enough for what it aims to exercise.